src/js/components/settings/__snapshots__/integrations.test.js.snap

```diff
@@ -35,15 +35,16 @@ exports[`Integrations Component renders correctly 1`] = `
         Azure IoT Hub connection string
       </label>
       <div
-        class="MuiInputBase-root MuiInput-root MuiInput-underline Mui-disabled Mui-disabled MuiInputBase-formControl MuiInput-formControl"
+        class="MuiInputBase-root MuiInput-root MuiInput-underline Mui-disabled Mui-disabled MuiInputBase-formControl MuiInput-formControl MuiInputBase-multiline MuiInput-multiline"
       >
-        <input
+        <textarea
           aria-invalid="false"
-          class="MuiInputBase-input MuiInput-input Mui-disabled Mui-disabled"
+          class="MuiInputBase-input MuiInput-input Mui-disabled Mui-disabled MuiInputBase-inputMultiline MuiInput-inputMultiline"
```

Comment:
You can also use the "multiline" property, which I think looks a little nicer https://mui.com/components/text-fields/#multiline